js/src/components/ads-account-select-control/index.js

```diff
@@ -10,8 +10,9 @@ import useExistingGoogleAdsAccounts from '.~/hooks/useExistingGoogleAdsAccounts'
  */
 const AdsAccountSelectControl = ( props ) => {
 	const { existingAccounts } = useExistingGoogleAdsAccounts();
+	const accounts = props.accounts || existingAccounts;
```

Comment:
@ankitrox The component should not receive any accounts prop. Instead the accounts are resolved via the `useExistingGoogleAdsAccounts` hook.

Reply:
If the accounts are resolved via `useExistingGoogleAdsAccounts` , and if there are more than one ads accounts, do you want me to use `disabled`  prop for `AdsAccountSelectControl` ?

Reply:
Handled it as mentioned in further comments.